rules/helm/package.sh

```diff
@@ -4,15 +4,18 @@ set -o errexit -o nounset
 
 build_dir="tmp/build/${PACKAGE_DIR}"
 mkdir -p "${build_dir}"
+
 cp -L -R "${PACKAGE_DIR}"/* "${build_dir}"
 
 for t in ${TARS}; do
-  tar xf "${t}" -C "${build_dir}"
+  tar xf "${t}" -C "${build_dir}"  > /dev/null
 done
 
+echo "operatorChartUrl: \"${OPERATOR_URL}\"" >> "${build_dir}"/Metadata.yaml
+
 "${HELM}" init --client-only > /dev/null
 "${HELM}" package "${build_dir}" \
   --version="${CHART_VERSION}" \
   --app-version="${APP_VERSION}" > /dev/null
 
-mv "${OUTPUT_FILENAME}" "${OUTPUT_TGZ}"
+mv "${OUTPUT_FILENAME}" "${OUTPUT_TGZ}"
```

Comment:
Nit: new line at EOF?

Reply:
Fixed in https://github.com/SUSE/kubecf/pull/149/commits/1e7d013029268c6cd42d44b0fd15ee66a270f3b0 (will squash eventually)